package.json

```diff
@@ -46,6 +50,7 @@
     "test": "npm-run-all test:*",
     "test:karma": "karma start",
     "test:lint": "eslint .",
-    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec"
+    "test:mocha": "istanbul cover _mocha --report lcovonly -- tests/*.test.js -R spec",
+    "clientize": "webpack"
```

Comment:
We probably don't need to have the artifact produced by the webpack command to live in the repo so we don't need to run this as part of a prepublish stage.